### PR TITLE
refactor(Aurelia): upgrade to Aurelia post beta usage

### PR DESCRIPTION
**Pull request by bigopon**
This PRs introduces an enhancement for Aurelia implementation:
- uses latest features/version
- uses progressive enhancement
- removes the need for bundler

### CODE REVIEW

**Comment by bigopon** [comment]
Thanks @FadySamirSadek I'll fix the auto focus issue.

About import SyntaxError, it's using es module syntax. Does that test mean you can only use es5?

**Comment by FadySamirSadek** [comment]
> About import SyntaxError, it's using es module syntax. Does that test mean you can only use es5?

Yes, but you can use babel to compile to es5 in your dist f

**Comment by FadySamirSadek** [comment]
@bigopon Any update?

**Comment by bigopon** [comment]
@FadySamirSadek yes, in fact i was updating it yesterday. Will push new commits soon. Thanks for the nudge ❤

**Comment by bigopon** [comment]
@FadySamirSadek it seems all tests for all frameworks are failing, I'm not sure how to resolve this 😥

**Comment by bigopon** [comment]
@FadySamirSadek 
> but if you're confident that you solution now works

Yes, i'm fairly confident I've met all of the requirements. I also took a small gif to show how it works on IE11 (progressive enhancement, es5, script tag):
todomvc

If you could help with testing and moving it forward, it would be really great!

File: examples/aurelia/.gitignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
+node_modules/*
```
Comment (FadySamirSadek) [inline examples/aurelia/.gitignore]: All TodoMVC apps should run without the user having to install npm packages or building anything so please remove node_modules from your .gitignore

Comment (FadySamirSadek) [reply examples/aurelia/.gitignore]: The exampled should be committed with all the files it needs to run

Comment (bigopon) [reply examples/aurelia/.gitignore]: @FadySamirSadek The following lines of that gitignore file should ensure user have all the necessary files to run the app, unless I misunderstood the git ignore syntax

Comment (FadySamirSadek) [reply examples/aurelia/.gitignore]: You're right, my bad

Comment (FadySamirSadek) [reply examples/aurelia/.gitignore]: LGTM

**Comment by bigopon** [comment]
@FadySamirSadek Awesome, thanks!

**Pull request merged**